### DISCUSSION TITLE
在不同设备登陆同一账号不会同步更新数据

### DISCUSSION THREAD

**Discussion by psy666** [ZuodaoTech/everyone-can-use-english#779] [Q&A]
用两台电脑登陆同一个账号，但是在前一台电脑生成的数据不会同步在第二台电脑显示，比如和AI的对话结果、已经生成过的音频。
请问目前是否就是这样的？能否新加一下同步功能？谢谢。
![Uploading 1.png…]()

**Comment by an-lee** [comment]
主要数据都是保存在本地的 EnjoyLibrary 里，目前还没有同步的开发计划。

不过你可以利用百度网盘来同步，将 EnjoyLibrary 放进网盘的同步列表里就可以了。

https://1000h.org/enjoy-app/settings.html#%E8%B5%84%E6%BA%90%E5%BA%93%E4%BF%9D%E5%AD%98%E8%B7%AF%E5%BE%84

**Comment by psy666** [comment]
了解了，非常感谢

**Reply by chenfx2033** [reply]
我用笔记本电脑生成了一下视频，在笔记本电脑上应用都挺正常。 我把EnjoyLibrary 目录全部copy到台式计算机上，登录的同一个用户，也设置了软件的资源库保存路径。  但在软件里面不显示我在笔记本电脑上已经添加的视频。
![image](https://github.com/user-attachments/assets/74a41b72-9a5b-4b1e-b12d-3739970ea101)

**Comment by chenfx2033** [comment]
用笔记本的enjoy_database.sqlite覆盖台式机上的文件，就ok了。
